src/Sulu/Bundle/SecurityBundle/Resources/translations/admin.de.json

```diff
@@ -7,6 +7,7 @@
     "sulu_security.system": "System",
     "sulu_security.system_language": "Systemsprache",
     "sulu_security.reset_mail_subject": "Sulu Password zurücksetzen",
+    "sulu_security.reset.mail_message": "Klicken Sie auf den folgenden Link oder kopieren Sie ihn in Ihren Browser um Ihr Sulu Passwort zurückzusetzen:",
```

Comment:
I think the key should be `sulu_security.reset_mail_message` in order to be consistent with `sulu_security.reset_mail_subject`.